SPARCbot.py

```diff
@@ -168,8 +184,276 @@ async def iwantto(self, ctx, role: str):
     async def unsure(self, ctx):
         await timed_send(ctx, 'I\'m afraid I can\'t let you do that.')
 
+class Bets(commands.Cog):
 
 
+    def __init__(self,bot):
+        self.bot = bot
+        self.lock = False
+
+    def add_new_bet(self,name,statement,status='open'):
```

Comment:
code style in function calls (here and elsewhere in this file): `add_new_bet(self, name, statement, status='open')` --- spaces after commas but not around the `=`

Reply:
Done